examples/booleans/booleans.md

```diff
@@ -1,8 +1,6 @@
 # Booleans and conditionals
 
-The `boolean` type has two values: `true`, `false`.
-The `!` operator works on booleans only.  `&&` and `||` operators short-circuit as in C.
-Usual comparison operators (`==`, `!=`, `<`, `>`, `<=`, and `>=`) produce boolean values.
+The `boolean` type has two values: `true`, `false`. The `!` operator works on booleans only. `&&` and `||` operators short-circuit as in C. Usual comparison operators (`==`, `!=`, `<`, `>`, `<=`, and `>=`) produce boolean values.
```

Comment:
```suggestion
The `boolean` type has two values: `true`, `false`. The `!` operator works only on booleans. `&&` and `||` operators short-circuit as in C language. Usual comparison operators (`==`, `!=`, `<`, `>`, `<=`, and `>=`) produce boolean values.
```
The idea of this sentence is not clear: ``&&` and `||` operators short-circuit as in C`